marklogic-data-hub/src/main/java/com/marklogic/hub/impl/HubConfigImpl.java

```diff
@@ -116,6 +116,9 @@
     private String hubAdminRoleName = DEFAULT_ADMIN_ROLE_NAME;
     private String hubAdminUserName = DEFAULT_ADMIN_USER_NAME;
 
+    //we assume DHF 2.x unless otherwise told, our earliest 'from' version
+    private String DHFVersion = "2.0.0";
```

Comment:
this is repeated code somewhere else.   I guess the DHF Version may very well be different from jarVersion?  but why would that be.  This smells I guess